src/main/java/org/folio/marccat/dao/persistence/T_AUT_BLNGL_USG.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.marccat.dao.persistence;
+
+/**
+ * @author elena
+ *
+ */
+public class T_AUT_BLNGL_USG extends T_SINGLE_CHAR {
```

Comment:
Can you please explain the purpose of these empty classes?  Also, is there a reason they need to be named like this?  FOLIO has adopted the pattern of naming classes with CamelCase.  See sonarqube.

Reply:
I have a question on this point for Craig McNally. The name corresponds to that of the table. We have many tables in mod-marccat. In particular, the T_s are decoding tables. It is preferable to leave them with this name for greater readability. I saw that sonarqube reports them. Should they all be changed, what do you suggest or do you suggest to leave them like this?